pkg/common/util.go

```diff
@@ -74,3 +75,8 @@ func Exist(name string) bool {
 	_, err := os.Stat(name)
 	return err == nil
 }
+
+// SplitVersionNum split version from release or tag
+func SplitVersionNum(release string) string {
```

Comment:
Usually, we use split to split a string into an array. How about change the name to `ParseVersionNum`